### DISCUSSION TITLE
too many requests

### DISCUSSION THREAD

**Discussion by pepijndevos** [TheR1D/shell_gpt#385] [Q&A] [answered]
I'm hitting https://github.com/TheR1D/shell_gpt/issues/201 but none of the options seem to apply

- You ran out of free credit, check your [dashboard](https://platform.openai.com/account/usage). Plenty of credit I'd say:
   ![image](https://github.com/TheR1D/shell_gpt/assets/168609/774239b8-581a-45e4-8593-c955276dc08b)
- You are hitting a rate [limit](https://help.openai.com/en/articles/5955604-how-can-i-solve-429-too-many-requests-errors). This is literally the only GPT API call I did today.
-  OpenAI servers are down, check the [status page](https://status.openai.com/). Status is green.

Anything else that could be a problem?

**Comment by TheR1D** [comment, marked as answer]
The free 18$ credits OpenAI provided during API launch are expired. Currenly it is required to link credit/debit card in order to make API requests.

**Reply by pepijndevos** [reply]
Ah ok thanks, would be good to update the linked issue to reflect that.
